Dockerfile

```diff
@@ -1,46 +1,43 @@
 ############################
 # STEP 1 build executable binary
 ############################
-FROM golang:1.13-alpine AS builder
-
-RUN apk update && apk add --no-cache git
-
-ARG COMMIT_SHA=<not-specified>
-
-# Create appuser.
-RUN adduser -D -g '' appuser
+FROM golang:1.14 AS builder
 
 WORKDIR /app
-COPY . .
 
-RUN echo "$COMMIT_SHA" >> ./commit.sha
+COPY go.mod .
```

Comment:
These are already copied at line 14, no need to be explicit on them

Reply:
This line is needed in order to use the docker cache during build. Copying all the repository, we lose the cache downloading the modules